packages/lit-dev-content/package.json

```diff
@@ -7,23 +7,136 @@
-    "build": "npm run build:not-site && npm run build:site",
-    "build:not-site": "npm run build:ts && npm run build:rollup && npm run build:samples",
-    "build:site": "rm -rf _site && LITDEV_ENV=local NODE_OPTIONS=--experimental-vm-modules eleventy",
-    "build:site:watch": "npm run build:site -- --watch",
-    "build:ts": "../../node_modules/.bin/tsc",
-    "build:ts:watch": "../../node_modules/.bin/tsc --watch",
-    "build:rollup": "rm -rf rollupout && rollup -c",
-    "build:rollup:watch": "rm -rf rollupout && rollup -c --watch",
-    "build:samples": "rm -rf samples/js && node ../lit-dev-tools-esm/lib/generate-js-samples.js",
-    "build:samples:watch": "npm run build:samples -- --watch",
-    "dev:build": "npm run build:not-site && npm run dev:build:site",
-    "dev:build:site": "rm -rf _dev && LITDEV_ENV=dev NODE_OPTIONS=--experimental-vm-modules eleventy",
-    "dev:build:site:watch": "npm run dev:build:site -- --watch --incremental",
-    "dev:serve": "node ../lit-dev-tools-esm/lib/dev-server.js --open",
-    "prod:build": "npm run build:not-site && NODE_OPTIONS=--experimental-vm-modules eleventy"
+    "fonts:manrope": "wireit",
+    "build": "wireit",
+    "build:not-site": "wireit",
+    "build:site": "wireit",
+    "build:site:watch": "wireit",
+    "build:ts": "wireit",
+    "build:ts:watch": "npm run build:ts -- watch",
+    "build:rollup": "wireit",
+    "build:rollup:watch": "npm run build:rollup -- watch",
+    "build:samples": "wireit",
+    "build:samples:watch": "npm run build:samples -- watch",
+    "dev:build": "wireit",
+    "dev:build:site": "wireit",
+    "dev:build:site:watch": "wireit",
+    "dev:serve": "wireit",
+    "prod:build": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:site"
+      ]
+    },
+    "dev:build": {
+      "dependencies": [
+        "dev:build:site"
+      ]
+    },
+    "prod:build": {
+      "dependencies": [
+        "build:not-site"
+      ],
+      "command": "NODE_OPTIONS=--experimental-vm-modules eleventy"
+    },
+    "dev:serve": {
+      "dependencies": [
+        "../lit-dev-tools-esm:build",
+        "dev:build"
+      ],
+      "command": "node ../lit-dev-tools-esm/lib/dev-server.js --open"
+    },
+    "build:site": {
+      "dependencies": [
+        "build:not-site"
+      ],
+      "output": [
+        "_site"
+      ],
+      "command": "LITDEV_ENV=local NODE_OPTIONS=--experimental-vm-modules eleventy"
+    },
+    "build:site:watch": {
+      "dependencies": [
+        "build:not-site"
+      ],
+      "output": [
+        "_site"
+      ],
+      "#comment": "This is awkward. We want to do the watch part for build:not-site ourselves, but let eleventy do its own watch mode itself.",
```

Comment:
I wonder if `--incremental` works outside of watch mode. In that case there might be no advantage to using the built-in watch mode. The docs are unclear: https://www.11ty.dev/docs/usage/incremental/. Also curious about interaction with `clean: if-file-deleted`. 

I'm fine with not thinking about that for this PR, though.